eng/targetframeworksuffix.props

```diff
@@ -9,6 +9,8 @@
     <TargetPlatformIdentifier>$(TargetFrameworkSuffix)</TargetPlatformIdentifier>
     <TargetPlatformVersion>1.0</TargetPlatformVersion>
     <TargetPlatformMoniker>$(TargetFrameworkSuffix),Version=$(TargetPlatformVersion)</TargetPlatformMoniker>
+    <!-- Value of 0.0 produces versionless SupportedOSPlatform attribute, which is important for platforms not expected to have a version -->
+    <SupportedOSPlatformVersion>0.0</SupportedOSPlatformVersion>
```

Comment:
Just to confirm -- are there any platforms where we _do_ want to have versioned attributes?

Reply:
I don't think so, for now, all targets are unversioned, if we ever need a versioned attribute that should be coming from the TFM suffix